doc/design/upgrade.md

```diff
@@ -1,20 +1,37 @@
 # Upgrade
 
-After installed VCH, VIC admin would want to upgrade existing instances to latest version. This document describes the upgrade requirement, and design options.
+After installed VCH, VIC admin would want to upgrade existing instances to latest version, to get new features or to fix exposed security issues. This document describes the upgrade requirement, and design options.
 
 ## Requirements
 
 ### Scope
 
-While a new version is ready, new version's vic-machine upgrade can upgrade one running or powered off VCH appliance. The containers managed by this VCH will be left in existing version. To have a new version container, user need to create new container from upgraded VCH instance. If existing container is stopped and restarted, the old version should be kept.
+While a new version is ready, new version's vic-machine can upgrade one running or powered off VCH appliance.
```

Comment:
hmm... It's actually saying even appliance is powered off, vic-machine still can upgrade it. Will update